packages/daimo-api/src/env.ts

```diff
@@ -1,5 +1,10 @@
 import { DaimoChain, getChainConfig } from "@daimo/contract";
 
+// Enable JSON.stringify for BigInts
```

Comment:
i'm very dubious of these kinds of global edits, but in this case i think it's OK

Reply:
Agree - I think in this case there's a great chance of bugs being caused by console logs/warnings/errors crashing due to lack of this rather than the other way around.